CMR-10455: Update subscription doc on subscription edge case limit (#2236)
nasa/Common-Metadata-Repository: ingest-app/docs/api.md

        </ul>
</ul>

> **IMPORTANT: There is limit to creating and updating subscriptions related to the specific scenario highlighted below:

Reviewer: ```suggestion
**IMPORTANT:** There is limit to creating and updating subscriptions related to the specific scenario highlighted below:
```

> Currently, if User1 creates a subscription to Coll1 and then User2 creates a different subscription to the same Coll1 for the same queue, it will overwrite the first subscription for that specific queue.

Reviewer 2: User1 rubs me the wrong way, only because people complain about our API docs a lot. While I am probably responsible for a good number of documentation complaints (having created a lot of them) I feel like we could do something better here. What about `"<User One>"` and `"<Collection One>"`?

Reviewer 2: I also do not like the use of ** for markdown when it has no markdown meaning. This could confuse bad parsers.

> Currently, if User1 creates a subscription to Coll1 and then User2 creates a different subscription to the same Coll1 for the same queue, it will overwrite the first subscription for that specific queue.

Reviewer: ```suggestion
Currently, if <User One> creates a subscription to <Collection 1> and then <User 2> creates a different subscription to the same <Collection 1> for the same queue, it will overwrite the first subscription for that specific queue.
```

> So please have each of your queues to have only one agreed upon subscription.

Reviewer 2: is this the correct "voice" for documentation, the use of "your"?

> So please have each of your queues to have only one agreed upon subscription.

Reviewer: ```suggestion
So please, have each queue have only one agreed upon subscription.
```

PUT requests should be used for updating subscriptions. Creating subscriptions with PUT may be deprecated in the future. All PUT requests require a native-id to be part of the request URL.

> **IMPORTANT: There is limit to creating and updating subscriptions related to the specific scenario highlighted below:

Reviewer 2: all the same comments from above.

> **IMPORTANT: There is limit to creating and updating subscriptions related to the specific scenario highlighted below:

Reviewer: ```suggestion
**IMPORTANT:** There is a limit to creating and updating subscriptions related to the specific scenario highlighted below:
```

> Currently, if User1 creates a subscription to Coll1 and then User2 creates a different subscription to the same Coll1 for the same queue, it will overwrite the first subscription for that specific queue.

Reviewer: ```suggestion
Currently, if <User 1> creates a subscription to <Collection 1> and then <User 2> creates a different subscription to the same <Collection 1> for the same queue, it will overwrite the first subscription for that specific queue.
```
